modules/configuring-the-distributed-workloads-components.adoc

```diff
@@ -0,0 +1,194 @@
+:_module-type: PROCEDURE
+
+[id="configuring-the-distributed-workloads-components_{context}"]
+= Configuring the distributed workloads components
+
+[role='_abstract']
+To configure the distributed workloads feature for your data scientists to use in {productname-short}, you must enable several components.
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+
+* You have access to the data science cluster.
+* You have installed {productname-long}.
+* You have sufficient resources. In addition to the base {productname-short} resources, you need 1.6 vCPU and 2 GiB memory to deploy the distributed workloads infrastructure.
```

Comment:
SME question - how do they find out what the base resources are?

Reply:
I've updated to link to the Installation Guide section.